SpatialGDK/Source/SpatialGDK/Private/Schema/UnrealObjectRef.cpp

```diff
@@ -133,6 +133,61 @@ FUnrealObjectRef FUnrealObjectRef::FromObjectPtr(UObject* ObjectValue, USpatialP
 	return ObjectRef;
 }
 
+FUnrealObjectRef FUnrealObjectRef::FromSoftObjectPtr(const FSoftObjectPtr& ObjectPtr)
+{
+	return FromSoftObjectPath(ObjectPtr.ToSoftObjectPath());
+}
+
+FUnrealObjectRef FUnrealObjectRef::FromSoftObjectPath(const FSoftObjectPath& ObjectPath)
+{
+	FUnrealObjectRef PackageRef;
+
+	PackageRef.Path = ObjectPath.GetLongPackageName();
+
+	FUnrealObjectRef ObjectRef;
+	ObjectRef.Outer = PackageRef;
+	ObjectRef.Path = ObjectPath.GetAssetName();
+
+	return ObjectRef;
+}
+
+void FUnrealObjectRef::ToSoftObjectPtr(const FUnrealObjectRef& ObjectRef, FSoftObjectPtr& OutPtr)
```

Comment:
Just wondering why this is made to be a static function? (taking in a const FUnrealObjectRef& when the function exists on FUnrealObjectRef)

Reply:
True, still following the trend of the other FromObjectPtr/ToObjectPtr, no other reason.